### ISSUE TITLE
add image upload functionality

### ISSUE THREAD

**Issue by jacksingleton** [jacksingleton/hacker-slides#15]

**Comment by zarvox** [comment]
We'll also need to modify reveal.js's CSS themes, since they currently load Google web fonts.  Perhaps add a build step that pre-fetches the various font files and adjusts the .scss to use the locally-cached copies?

**Comment by ndarilek** [comment]
Would be nice to allow uploading of more than just images. It might be useful to upload video, audio, etc. Even if off-site embeds are allowed, I might want my presentation to be entirely self-contained so Sandstorm's own backup/restore mechanisms capture the whole thing.

**Comment by jacksingleton** [comment]
Agreed @ndarilek. Eventually I think Hacker Slides should integrate with a file upload app like Filedrop or Davros via the powerbox and allow any file to be embedded within a presentation.

**Comment by JamborJan** [comment]
Sounds excellent @jacksingleton . This would be the best solution for the sandstorm environment.

**Comment by zeigerpuppy** [comment]
For the moment, there's a hacky way to do this with Davros
1.   upload image to Davros
2.   view image in Davros and copy the link from “Download Original”
   eg: 
   `<img src="https://16290b47747ab678fb611bf4741ss.yoursever.net/remote.php/webdav/tutorials/images/voyager.jpg" width=300>`
3. include this link in Hackerslides, and size to fit
